apps/api/src/server/applications/subscriptions/subscriptions.js

```diff
@@ -163,6 +171,30 @@ export function subscriptionTypeChanges(existingSub, newSub) {
 	return byType;
 }
 
+/**
+ * @param {import('@pins/applications.api').Schema.Subscription} subscription
+ * @return {ServiceUser}
+ * */
+export const buildServiceUserPayload = (subscription) => ({
+	id: subscription.serviceUser.id.toString(),
+	firstName: subscription.serviceUser.firstName,
+	lastName: subscription.serviceUser.lastName,
+	addressLine1: subscription.serviceUser.address?.addressLine1,
+	addressLine2: subscription.serviceUser.address?.addressLine2,
+	addressTown: subscription.serviceUser.address?.town,
+	addressCounty: subscription.serviceUser.address?.county,
+	postcode: subscription.serviceUser.address?.postcode,
+	addressCountry: subscription.serviceUser.address?.country,
+	organisation: subscription.serviceUser.organisationName,
+	role: subscription.serviceUser.jobTitle,
+	telephoneNumber: subscription.serviceUser.phoneNumber,
+	emailAddress: subscription.serviceUser.email,
+	serviceUserType: 'Subscriber',
+	caseReference: subscription.caseReference,
+	sourceSuid: subscription.serviceUser.id.toString(),
+	sourceSystem: 'BO'
+});
```

Comment:
I think that would be better but we would want to refactor the code in `src/server/applications/application/representations/publish/representation.js` as we're doing something similar there. It might be better to do this as a separate tech task.

Reply:
Agreed, let's do that separately then. Could we add a ticket somewhere?

Reply:
Added a ticket to the backlog https://pins-ds.atlassian.net/browse/BOAS-1420